locations/spiders/les_petits_chaperons_rouges_fr.py

```
@@ -0,0 +1,21 @@
+
+from scrapy.http import TextResponse
+from scrapy.spiders import SitemapSpider
+
+from locations.categories import Categories, apply_category
+from locations.items import Feature
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class LesPetitsChaperonsRougesFRSpider(SitemapSpider, StructuredDataSpider):
+    name = "les_petits_chaperons_rouges_fr"
+    item_attributes = {"brand": "Les Petits Chaperons Rouges", "brand_wikidata": "Q130214582"}
```

Ruff (0.16.2)

[warning] Mutable default value for class attribute

(RUF012)